### ISSUE TITLE
The data set code seems to be out of date (and likely not functioning)

### ISSUE THREAD

**Issue by searchivarius** [craston/MARS#30]
Hi, thank you very much for publishing the code! 

I think the data set code might be outdated, though. Here, it uses the variable, which is not defined:

https://github.com/craston/MARS/blob/ae2749dc37ec19821da69654e9197a52f0b45839/dataset/dataset.py#L255

There's an obvious fix for this, but I now I wonder if you might have published not the latest and greatest version of the code.

Furthermore, the code that parses annotation files seems to be wrong as well. 

First, do you use **action detection **annotations** from [here](https://www.crcv.ucf.edu/research/data-sets/ucf101/)?

If this is the case, each line has the format:
<file name> <white space> <class id>
or 
<file name>
In other words, there's no class ID for test entries, but there's a class ID for training.

Thus, the following two lines fail silently for training entries.
https://github.com/craston/MARS/blob/ae2749dc37ec19821da69654e9197a52f0b45839/dataset/dataset.py#L264

**Comment by craston** [comment]
Hi, 

Thanks for pointing out the bug in the following line (It should be self.train_val_test) https://github.com/craston/MARS/blob/ae2749dc37ec19821da69654e9197a52f0b45839/dataset/dataset.py#L255
The code was published 1.5 years ago and since I switched organizations, I haven't been updating it.

As for the action recognition annotations, they were dowloaded from [here](https://www.crcv.ucf.edu/data/UCF101.php) (**I did not use the revised annotations)

**Comment by searchivarius** [comment]
Ohh, I see. Thank you for the clarifications. It was not a big deal to adapt it to the updated annotations. Thanks again for publishing the code!
